packages/design-system/package.json

```diff
@@ -63,9 +63,15 @@
     "@accelint/biome-config": "workspace:*",
     "@accelint/typescript-config": "workspace:*",
     "@accelint/vitest-config": "workspace:*",
+    "@chromatic-com/storybook": "^3.2.3",
     "@fontsource/roboto-flex": "5.1.0",
     "@fontsource/roboto-mono": "5.1.0",
-    "@ladle/react": "4.1.2",
+    "@storybook/addon-actions": "7.6.20",
```

Comment:
Actions is already part of essentials

Reply:
ts is unhappy if we try to import from this package without specifying it explicitly as a dep.
![Screenshot 2025-01-15 at 8 35 15 AM](https://github.com/user-attachments/assets/fdf751db-35ce-486a-af43-a41f32e674cb)